backend/src/test/service/CachingService.test.ts

```diff
@@ -0,0 +1,41 @@
+import { CachingService } from '../../main/service/CachingService';
+import { CachingObject } from '../../main/data/CachingObject';
+
+describe('simple add/get for redis cache', function () {
+  // NOTE: RUNNING THESE TESTS WILL MODIFY THE REDIS SERVER!
+  let cachingService: CachingService;
+
+  beforeEach(() => {
+    cachingService = new CachingService();
+  });
+
+  afterEach(() => {
+    const client = cachingService.getClient();
+    client.quit();
```

Comment:
I've left the Redis server open for the application since it takes resources to open- but open to more research and discussion as well 😄